### DISCUSSION TITLE
Custom field dynamic group condition

### DISCUSSION THREAD

**Discussion by kevin-vanb** [liedekef/events-made-easy#89] [Q&A] [answered]
Hi all,
I created a custom field (jersey size) as a member field.
Now I want a dynamic group so we can address only those members who did not fill in their size. (with a reminder mail)
The custom field is a dropdown field, so I tried to check on empty, null, '',"" and lots of variatons on these but nothing worked so far.

Any ideas?

Kind regards,
Kevin

**Comment by liedekef** [comment, marked as answer]
I checked the code, and currently the search only works if the value to search for is not empty, I never took into account the fact that people want to look for empty things ...
This should help: https://github.com/liedekef/events-made-easy/commit/f23e5f5240a7733aeef4ac9348206861fc417863

**Reply by kevin-vanb** [reply]
Thanks Franky, reason for asking is because we implemented a member field for existing members and we would like to send out a reminder for those members who did not yet changed their jersey size.
But small question, I copied the changed lines (deleted and inserted), but still receive an error.
Is this the best way to do so or is it better to download the files somehow?

kind regards,
Kevin

**Comment by liedekef** [comment]
An error? And what would the error be? Maybe I made some typo ... I'll test it here too
Ok, fixed it. Missing ";":
https://github.com/liedekef/events-made-easy/commit/9ac0bcfdc5a9b29200caac84a80c31e6663bdfeb

**Reply by kevin-vanb** [reply]
Great work Franky, thanks!

**Reply by liedekef** [reply]
The code had an issue (it was resulting in a too strict sql in some cases). I just released a version that fixes that.

**Reply by kevin-vanb** [reply]
Thanks Franky, much appreciated!
